### DISCUSSION TITLE
Weighted Burgers vector implementation question

### DISCUSSION THREAD

**Discussion by JasonOttUW** [mtex-toolbox/mtex#1924] [Ask Anything]
I just had a clarification question about the implementation of the weighted Burgers vector in MTEX. Are the resulting vectors given in units such as micrometers^(-1), or are they presented per unit (per pixel) on the EBSD map, such that we need to use the step size to get the actual units?
Thanks in advance!
Jason Ott

**Comment by ralfHielscher** [comment]
Hi Jason,

unfortunately, the weighted Burgers vector is currently per pixel.  I think we need to change this in the future.

Ralf.

**Comment by JasonOttUW** [comment]
Hello, Ralf,

Thanks for the response. I was looking through the code, and thought this
was the case, but just wanted to make sure I wasn't missing anything. But,
now that I know, , I can at least correct for that in my script using the
step size from the data

Thank you for the clarification,

Jason
